AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -749,16 +755,37 @@ - (BOOL)shouldBatchFetch
   }
 }
 
-- (void)handleBatchFetchScrollingToOffset:(CGPoint)targetOffset
+- (void)_scheduleCheckForBatchFetching
 {
-  ASDisplayNodeAssert(_batchContext != nil, @"Batch context should exist");
-  
-  if (![self shouldBatchFetch]) {
+  // Push this to the next runloop to be sure the UITableView has the right content size
+  dispatch_async(dispatch_get_main_queue(), ^{
+    [self _checkForBatchFetching];
+  });
+}
+
+- (void)_checkForBatchFetching
+{
+  // Dragging will be handled in scrollViewWillEndDragging:withVelocity:targetContentOffset:
+  if (self.isDragging || self.isTracking) {
```

Comment:
Out of curiosity, because I'm not sure, is it important to check for dragging and to not allow the batch to be triggered if the drag is in progress? It seems like if the drag does reached the threshold, we should trigger the load immediately at the point that it does reach it